### PR TITLE
chore(gen): documentation review MTA-5103

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: console/account/how-to/recover-a-lost-password.mdx [hunk 1]
```diff
@@ -19,29 +19,30 @@ If you forget or lose the password to your [Scaleway account](https://console.sc
...
-1. Go to the [lost password](https://console.scaleway.com/forgot-password/) page of the console:
+1. From the [Scaleway console login page](https://account.scaleway.com/), Click **Log in with password**.
+1. Click **Forgot your password?**, or access the [lost password](https://console.scaleway.com/forgot-password/) page.
 2. Enter your email address and click **Reset password**. A password reset link is sent to your email address.
 3. Click the link in the email address to reset your password.
 4. Enter your new password and click **Reset password** to set the new password for your account. You are redirected to the Login page.
 5. Log in to the console with your email address and your new password.
 
-## Password recovery in case of lost email account
+## Recover your password if you lost your email account
```
Comment (bene2k1) [inline console/account/how-to/recover-a-lost-password.mdx:41]:
```suggestion
## Recover your password if you lost access to your email account
```
wdyt? Not sure if this is too long

File: faq/serverless-sql-databases.mdx [hunk 2]
```diff
@@ -14,7 +14,7 @@ category: serverless
 
 You are billed both for the compute resources provisioned and the storage used. You can see the cost estimate in the Database creation wizard.
 
-  - **Compute**: You are billed based on compute resources (vCPU and RAM) provisioned to handle your workload. Provisioned resources evolve according to the [autoscaling](/serverless/sql-databases/reference-content/serverless-sql-databases-overview/#autoscaling) parameter and stay between the minimum and maximum thresholds defined for your database. For each vCPU provisioned, 4 GB of RAM will be allocated.
+  - **Compute**: You are billed based on compute resources (vCPU and RAM) provisioned to handle your workload. Provisioned resources evolve according to the [autoscaling](/serverless/sql-databases/reference-content/serverless-sql-databases-overview/#autoscaling) parameter, and stay between the minimum and maximum thresholds defined for your database. For each vCPU provisioned, 4 GB of RAM will be allocated.
```
Comment (bene2k1) [inline faq/serverless-sql-databases.mdx:17]:
```suggestion
  - **Compute**: You are billed based on compute resources (vCPU and RAM) provisioned to handle your workload. Provisioned resources evolve according to the [autoscaling](/serverless/sql-databases/reference-content/serverless-sql-databases-overview/#autoscaling) parameter, and stay between the minimum and maximum thresholds defined for your database. For each vCPU provisioned, 4GB of RAM will be allocated.
```

**Review by bene2k1 — APPROVED** [review]

File: faq/serverless-jobs.mdx [hunk 3]
```diff
@@ -84,19 +84,18 @@ Serverless Jobs are integrated with [Cockpit](/observability/cockpit/quickstart/
...
 
 ## How can I automate the deployment and management of Scaleway Serverless Jobs?
 
 Scaleway Serverless Jobs is part of the Scaleway ecosystem, it can therefore be driven using the [Scaleway CLI](/developer-tools/scaleway-cli/quickstart/), the [Scaleway API](https://www.scaleway.com/en/developers/api/), and other [developer tools](https://www.scaleway.com/en/developers/). Our serverless ecosystem offers a lot of possibilities with event-driven architectures, and integrations with more products of the Scaleway ecosystem are under active development.
 
-## Do Serverless Jobs offer parallelization?
+## Can I run multiple Serverless Jobs at the same time?
 
-Not yet. Scaleway Serverless Jobs will soon offer parallelization via a `parallelism` parameter for each job. This will automatically launch a given number of replicas of the Job container in parallel.
+When starting a Job, you can use contextual options to define the number of jobs to execute at the same time. Refer to the [dedicated documentation](/serverless/jobs/how-to/run-job/#how-to-run-a-job-with-contextual-options) for more information.
```
Comment (RoRoJ) [inline faq/serverless-jobs.mdx:95]: not sure about capitalization of job at start of sentence and then not at the second mention...but it's a minefield...

File: faq/serverless-sql-databases.mdx (hunk 2 above)
Comment (RoRoJ) [reply faq/serverless-sql-databases.mdx:17]: The original is right according to the style guide https://www.scaleway.com/en/docs/guidelines/#bits-and-bytes

File: serverless/containers/how-to/add-a-custom-domain-to-a-container.mdx [hunk 4]
```diff
@@ -81,15 +81,15 @@ HTTP-01 challenge failure (and by extension, a custom domain in `error` status)
...
-| Route `/.well-known/acme-challenge` is not handled correctly. | To perform the HTTP-01 challenge, a call will be made on `<your_custom_domain>/.well-known/acme-challenge/<token>`. If you are using a proxying/caching tool (like Cloudflare) between the custom domain and the Serverless Container endpoint, a cached version of that route can be served, making the check fails. | Don't configure caching rules for routes starting with `/.well-known/acme-challenge`. See an example below with Cloudflare. |
+| Route `/.well-known/acme-challenge` is not handled correctly. | To perform the HTTP-01 challenge, a call will be made on `<your_custom_domain>/.well-known/acme-challenge/<token>`. If you are using a proxying/caching tool (like Cloudflare) between the custom domain and the Serverless Container endpoint, a cached version of that route can be served, making the check fails. | Do not configure caching rules for routes starting with `/.well-known/acme-challenge`. Rerfer to the example below with Cloudflare. |
```
Comment (RoRoJ) [inline serverless/containers/how-to/add-a-custom-domain-to-a-container.mdx:84]:
```suggestion
| Route `/.well-known/acme-challenge` is not handled correctly. | To perform the HTTP-01 challenge, a call will be made on `<your_custom_domain>/.well-known/acme-challenge/<token>`. If you are using a proxying/caching tool (like Cloudflare) between the custom domain and the Serverless Container endpoint, a cached version of that route can be served, making the check fails. | Do not configure caching rules for routes starting with `/.well-known/acme-challenge`. Refer to the example below with Cloudflare. |
```

**Review by RoRoJ — APPROVED** [review]

File: faq/serverless-jobs.mdx (hunk 3 above)
Comment (SamyOubouaziz) [inline faq/serverless-jobs.mdx:95]:
```suggestion
When starting a job, you can use contextual options to define the number of jobs to execute at the same time. Refer to the [dedicated documentation](/serverless/jobs/how-to/run-job/#how-to-run-a-job-with-contextual-options) for more information.
```

**Pull request merged**